Library/Homebrew/cask/cmd/brew-cask-tests.rb

```diff
@@ -52,6 +52,9 @@ def run_tests(executable, files, args = [])
 
   if upload_coverage
     puts "Submitting Codecov coverage..."
-    system "bundle", "exec", "rake", "test:coverage:upload"
+    require "simplecov"
+    require "codecov"
+    formatter = SimpleCov::Formatter::Codecov.new
+    formatter.format(SimpleCov::ResultMerger.merged_result)
```

Comment:
@reitermarkus Check the Travis CI output; this doesn't work how I'd thought it would. Thoughts to avoid shelling out? Alternatively, thoughts on just making a standalone script for this rather than a Rakefile (although I'd love to get it in-process).